internal/dao/registry.go

```diff
@@ -117,6 +118,77 @@ func AccessorFor(f Factory, gvr client.GVR) (Accessor, error) {
 	return r, nil
 }
 
+func GVRForKind(apiVersion string, kind string) (string, bool, error) {
```

Comment:
An alternative would be to simply concatenate and hope for the best...
```go
apiVersion + "/" + strings.ToLower(kind) + "s"
```
Too risky?

Reply:
Indeed! I think there are only a handful of qualified controllers. I think we could just simply have a map kind->resource. Might be easier to maintain in the future should these go thru an api rev.

Reply:
Is this what you imagined?

https://github.com/derailed/k9s/blob/0edc12ffa4af9818157642b4e9f20dc7f2986fa8/internal/dao/registry.go#L131-L143

Reply:
I think the best way to handle getting to gvr/fqn in the helper is as follows:

```go
mapper := RestMapper{Connection: c}
m, err := mapper.ToRESTMapper()
rm, err := m.RESTMapping(schema.GroupKind{Group: g, Kind: k}, v)
gvr := client.NewGVR(rm.Resource.String())
meta, err := MetaAccess.MetaFor(gvr)
if meta.Namespaced {} else {}
```

Reply:
I got pretty close to your suggestion. The map is gone now thanks to the `RestMapper`.